docs/remote/devcontainer-cli.md

```diff
@@ -126,7 +138,75 @@ For example, you may want to pre-build several images that you then reuse across
 
 That's it!
 
-### Adding automation
+## Building Multi-Architecture builds
+
+#### Pre-requisites
+1. [Node.js 14+](https://nodejs.org).
+2. [The `docker` CLI](/docs/remote/containers#installation).
+3. [Docker `Buildx`](https://docs.docker.com/buildx/working-with-buildx/)
+4. [Docker logged in to a Dockerhub registry](https://docs.docker.com/engine/reference/commandline/login/)
+
+Ensure that the command `docker buildx -h` returns a valid help message.
+
+Docker needs to be logged in to a container registry because the multi-architecture build defaults to pushing the multi-arch image manifesto to a container registry. If not, the `--no-push` option needs to be used.
+
+#### Process Example
+
+1. Create a basic Dockerfile as below:
+
+    ```Dockerfile
+    ARG VARIANT=3.13
+    FROM mcr.microsoft.com/vscode/devcontainers/base:0-alpine-${VARIANT}
+    CMD echo "Running on $(uname -m)"
+    ```
+
+2. Create a basic `devcontainer.json` as below:
+
+    ```json
+    {
+    "name": "Alpine",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": { "VARIANT": "3.15" },
+        "settings": {},
+        "extensions": [],
+        "remoteUser": "vscode"
+      }
+    }
+    ```
+
+3. Place both files in a `.devcontainer` directory.
+
+4. Run the below command:
+
+    ```bash
+    devcontainer build \
+        --platform linux/amd64,linux/arm64 \
+        --image-name hubusername/multiarch-test:v1 \
+        path-to-devcontainer-json
+    ```
+
+#### Breaking down the command
+
+1. `devcontainer build`: As before, this invokes the devcontainer CLI build command.
+2. `--platform linux/amd64,linux/arm64`: This creates Docker images for the `linux/amd64` and `linux/arm64` architectures. Find other supported platforms [here](https://docs.docker.com/engine/install/) (all Docker supported platforms are supported) and [here](https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#platform) on the Docker Buildx documentation page.
+4. `--image-name hubusername/multiarch-test:v1`: **This is mandatory when using `--platform` and optional otherwise.** This will tag the image and needs to follow container registry naming conventions to upload.
```

Comment:
```suggestion
3. `--image-name hubusername/multiarch-test:v1`: **This is mandatory when using `--platform` and optional otherwise.** This will tag the image and needs to follow container registry naming conventions to upload.
```